content/logs/explore.md

```diff
@@ -179,6 +181,26 @@ If an attribute was called `user.first name`, perform a search on this attribute
 
 `@user.first\ name:myvalue`
 
+## Log Graph
+
+Switch between the Log List and the Log Graph mode by clicking on this button:
+
+{{< img src="logs/explore/graph/log_graph_switch.png" alt="Log graph switch" responsive="true" popup="true" style="width:80%;">}}
+
+The only available Log Graph (for now) is the timeseries:
+
+{{< img src="logs/explore/graph/basic_timeseries_log_graph.png" alt="basic timeseries log graph" responsive="true" popup="true" style="width:80%;">}}
+
+To start using it:
+
+1. Choose the Measure to graph
+2. Split by [Tag](/getting_started/tagging) or [Facet](#facets) to split your graph over the wanted dimension.
+{{< img src="logs/explore/graph/split_by_log_graph.png" alt="split by log graph" responsive="true" popup="true" style="width:80%;">}}
+
+3. See logs related to a section of the graph:  
+    Select or click on a section of the graph to either zoom in the graph or see the list of logs corresponding to your selection: 
```

Comment:
what is the difference between selecting and clicking?

Reply:
Selecting: You select a time range and see associated logs
Clicking: you click on an aggregation result to see the associated logs